src/Lesson2/Lesson2.java

```diff
@@ -0,0 +1,65 @@
+package Lesson2;
+
+public class Lesson2 {
+
+    public static void main(String[] args) {
+
+        checkSum(10, 3);
+        posOrNegWithPrint(-9);
+        posOrNegReturn(9);
+        printString("Privet", 8);
+        leapYearOrNot(2021);
+//      leapYearOrNottest(400);
+    }
+
+    public static boolean checkSum(int a, int b) {
+        if (10 >= (a + b) && (a + b) <= 20) {
+            return true;
+        }
+        return false;
+    }
+
+    public static void posOrNegWithPrint(int a) {
+        if (a >= 0) {
+            System.out.println("Положительное число");
+        } else
+            System.out.println("Отрицательное число");
+    }
+
+    public static boolean posOrNegReturn(int a) {
+        if (a < 0) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
       return a < 0;
```
можно упростить, поскольку логическое выражение само по себе является boolean значением на выходе: